pkg/infra/metrics/metrics.go

```diff
@@ -483,6 +489,18 @@ func init() {
 		Help:      "A metric with a constant '1' value labeled by pluginId, pluginType and version from which Grafana plugin was built",
 		Namespace: ExporterName,
 	}, []string{"plugin_id", "plugin_type", "version"})
+
+	StatsTotalDashboardVersions = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name:      "stat_totals_dashboard_versions",
```

Comment:
Why "stats_totals" instead of "stats_total"? stat_total would correspond to the variable name. Could it be a misplaced "s"?

Reply:
Follows same convention as earlier metrics like stat_totals_datasource and stat_totals_active_admins

Reply:
@marefr Yeah I saw it in retrospect. What's the reason for this naming convention differing from the variable names though?

Reply:
I dont think there is any thoughts about the convention. Just sticking to it to make sure it's the same. 

While I agree with you that it should be named `stats`, renaming old metrics is too much work, and not using the same pattern will misalign our graphite metrics. https://github.com/grafana/grafana/blob/master/pkg/infra/metrics/graphitebridge/graphite.go#L62

Reply:
Thanks for clarifying @bergquist.